test_runner/src/main/kotlin/ftl/environment/ListAndroidDevices.kt

```diff
@@ -0,0 +1,35 @@
+package ftl.environment
+
+import com.google.api.services.testing.model.AndroidModel
+import ftl.util.SystemOutColor
+import ftl.util.applyColorsUsing
+import ftl.util.buildTable
+
+fun List<AndroidModel>.asPrintableTable() = createTestEnvironmentInfo().createAndroidDevicesTable()
+
+private fun List<AndroidModel>.createTestEnvironmentInfo() =
+    fold(mutableMapOf<String, MutableList<String>>()) { devicesInfo, androidDevice ->
+        devicesInfo.apply {
+            getOrCreateList(MODEL_ID).add(androidDevice.codename)
```

Comment:
`codename`, `manufacturer`, `name`, `from` etc ... generally properties of `androidDevice` all are nullable values (java ... ). Sometimes there is no big harm but in other cases it may end up with NPE.
I am wondering if we could protect logic from `null`s and make flank more robust. I am aware it should not happen but I am also sure, sooner or later this will blow. 
Above is applicable to `ListAndroidSofwareVersions.kt`, `ListIOsDevices.kt`, `ListIOsSofwareVersions.kt` as well. (I just don't want to duplicate comment :) )

Let me know what do you think